cql3/CqlParser.g4

```diff
@@ -673,6 +680,10 @@ whereSpec
    : kwWhere relationElements
    ;
 
+jsonSpec
+   : kwJson
+   ;
```

Comment:
Maybe just inline the rule `jsonSpec`? I mean using the  token `K_JSON` everywhere instead of `jsonSpec`.

Reply:
You're right, simple keyword usage here.